code/modules/reagents/Chemistry-Recipes/Others.dm

```diff
@@ -124,6 +126,40 @@
 			D.RemoveSymptom(healer)
 			D.AddSymptom(new /datum/symptom/regen())
 			D.Refresh(1) //It needs to actually change the disease name for reasons
+		//scarab symptom
+		var/itching = locate(/datum/symptom/itching) in D.symptoms
+		var/hallucinogen = locate(/datum/symptom/hallucigen) in D.symptoms
+		var/vomit = locate(/datum/symptomvomit/blood) in D.symptoms
```

Comment:
Fix the typepath
code/modules/reagents/Chemistry-Recipes/Others.dm:132:error: /datum/symptomvomit/blood: undefined type path